widgetssdk/src/main/java/com/glia/widgets/core/chathead/ChatHeadManager.java

```diff
@@ -20,7 +24,24 @@ public ChatHeadManager(
     public void startChatHeadService() {
         if (!isServiceStarted) {
             isServiceStarted = true;
-            applicationContext.startService(chatHeadServiceIntent);
+            try {
+                applicationContext.startService(chatHeadServiceIntent);
+            } catch (IllegalStateException exception) {
+                isServiceStarted = false;
```

Comment:
Is there validity for notifying the user as well?

Reply:
@Pelkar Ideally, yes, it might be good. Do you have any ideas on how to notify?

Reply:
This could/should be discussed with Mari and Risto, but Risto is on vacation.

If I were to propose something, then either a general error dialog or a toast.

Reply:
What's the benefit of notifying the user about this? In my opinion, this is an internal error, and it should only be logged(most probably with a warning level). We should also take into account that the app should be in the background when this happens.

Reply:
@DavDo Regarding log level - I have changed the log level from info to warning.

Regarding dialog or toast, I'm not sure it would be easy for the case when app is in background for 60+ seconds.
Also, let's take into account that:
1) It's kind of an edge case - 12 cases per 90 days in Sentry
2) Once the visitor opens the app, bubble will start working

@Pelkar do you still think I need to raise a question to Mari and Risto?

Reply:
> Regarding dialog or toast, I'm not sure it would be easy for the case when app is in background for 60+ seconds.

Besides those difficulties, I am unsure if providing such information to the user is necessary.

Reply:
Fair enough if it is such an extreme edge case.